non_project_build.tcl

```diff
@@ -78,9 +85,11 @@ if { $synth_only } {
     exit
 }
 
-# STEP#4: run logic optimization, placement and physical logic optimization,
+# STEP#4: batch insert ila, run logic optimization, placement and physical logic optimization,
 # write design checkpoint, report utilization and timing estimates
 #
+source batch_insert_ila.tcl
```

Comment:
add a flag to control adding ILA or not

Reply:
`batch insert ila` is disabled by default now.